firmware/Core/Inc/boom_deploy_drivers/boom_deploy_drivers.h

```diff
@@ -0,0 +1,13 @@
+#ifndef INCLUDE_GUARD__BOOM_DEPLOY_DRIVERS_H
+#define INCLUDE_GUARD__BOOM_DEPLOY_DRIVERS_H
+
+
+#include "main.h"
```

Comment:
Why do you need to include main.h here? You already include it in the .c file, not sure why you need it here?

Reply:
Ah good catch, thanks